<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chessboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>80.000000000000000</width>
        <height>80.000000000000000</height>
       </rectf>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;FEN:</string>
        </property>
        <property name="buddy">
         <cstring>fenLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fenLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendPushButton">
        <property name="text">
         <string>&amp;Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Game"/>
    <addaction name="action_Connect"/>
    <addaction name="action_Disconnect"/>
    <addaction name="action_Request_Draw"/>
    <addaction name="action_Resign"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect...</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="action_New_Game">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
  </action>
  <action name="action_Request_Draw">
   <property name="text">
    <string>&amp;Request Draw</string>
   </property>
  </action>
  <action name="action_Resign">
   <property name="text">
    <string>&amp;Resign</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
